### PR TITLE
Update Theme

### PR DESCRIPTION
**Pull request by SerhiiTsybulskyi**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[x] Yes
[ ] No
```

## Other information

Screenshot 2024-08-01 at 16 33 04
Screenshot 2024-08-01 at 16 33 37
Screenshot 2024-08-01 at 16 33 55
Screenshot 2024-08-01 at 16 35 01

### CODE REVIEW

File: stories/Console.stories.tsx [hunk 1]
```diff
@@ -347,7 +349,11 @@ export const DefaultInputValue = () => {
               groups.map(({ heading, sessions }) => (
                 <SessionsGroup heading={heading} key={heading}>
                   {sessions.map(s => (
-                    <SessionListItem key={s.id} session={s} />
+                    <SessionListItem
+                      key={s.id}
+                      session={s}
+                      chatIcon={<ChatIcon className="mr-2" />}
```
Comment (amcdnl) [inline stories/Console.stories.tsx:355]: Why pass this ?

File: tailwind.config.ts [hunk 2]
```diff
@@ -11,6 +11,9 @@ module.exports = {
   ],
   theme: {
   extend: {
+    borderRadius: {
+      '3xl': '20px',
```
Comment (amcdnl) [inline tailwind.config.ts:15]: reablocks should have this no?

File: src/SessionsList/SessionListItem.tsx [hunk 3]
```diff
@@ -32,7 +37,8 @@ export const SessionListItem: FC<SessionListItemProps> = ({
   session,
   deletable = true,
   limit = 100,
-  deleteIcon = <TrashIcon />
+  deleteIcon = <TrashIcon />,
+  chatIcon
```
Comment (amcdnl) [inline src/SessionsList/SessionListItem.tsx:41]: Should initialize chat icon here probs

File: tailwind.config.ts (hunk 2 above)
Comment (SerhiiTsybulskyi) [reply tailwind.config.ts:15]: default value of 3xl is 24px

File: src/SessionsList/NewSessionButton.tsx [hunk 4]
```diff
@@ -1,32 +1,44 @@
...
 interface NewSessionButtonProps extends PropsWithChildren {
   /**
    * Text for the new session button.
    */
   newSessionText?: string | ReactNode;
+
+  /**
+   * Divider to render between the new session button and SessionsGroup.
+   */
+  divider?: ReactNode;
```
[the remark this answers is not in the document]
Comment (SerhiiTsybulskyi) [reply src/SessionsList/NewSessionButton.tsx]: moved to theme style

**Pull request merged**